README.md

```diff
@@ -4,7 +4,7 @@
 
 This repository contains the specs, docs and examples of the Devfile v1.
 
-For Devfile v2 go [here](https://devfile.github.io/devfile/index.html). 
+For Devfile v2 go [here](https://devfile.github.io). 
```

Comment:
```suggestion
For Devfile v2 go [here](https://docs.devfile.io). 
```

Reply:
You're probably right. I was going off `about` section here - the https://github.com/devfile/devfile.github.io. If they've settled on docs.devfile.io they should update that info too ;)